.github/workflows/solidity.yml

```diff
@@ -76,9 +106,10 @@ jobs:
         working-directory: contracts
         run: pnpm prepublishOnly
 
+  # TODO: Confirm if this job is needed for CCIP contracts releases.
```

Comment:
@RensR ^

Reply:
Confirmed that it does not need to run for CCIP contracts releases and is fine as-is.